src/components/app/data/hooks/useEnterpriseCustomer.test.jsx

```diff
@@ -1,53 +1,111 @@
 import { renderHook } from '@testing-library/react-hooks';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { AppContext } from '@edx/frontend-platform/react';
+import { MemoryRouter, useParams } from 'react-router-dom';
 import { authenticatedUserFactory, enterpriseCustomerFactory } from '../services/data/__factories__';
 import useEnterpriseCustomer from './useEnterpriseCustomer';
 import { queryClient } from '../../../../utils/tests';
-import { fetchEnterpriseLearnerData } from '../services';
+import { fetchEnterpriseLearnerDashboard, fetchEnterpriseLearnerData } from '../services';
 
 jest.mock('../services', () => ({
   ...jest.requireActual('../services'),
   fetchEnterpriseLearnerData: jest.fn().mockResolvedValue(null),
+  fetchEnterpriseLearnerDashboard: jest.fn().mockResolvedValue(null),
+}));
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(),
```

Comment:
Similar comment here about avoiding mocking `useParams` if we're rendering a real router via `MemoryRouter` below.